### ISSUE TITLE
Description

### ISSUE THREAD

**Issue by nurmians** [cancerit/alleleCount#48]
I would be helpful to have a description of what the software does and what it can be used for.

**Comment by bjpop** [comment]
I agree, a description of what the program does would be very helpful.

**Comment by drjsanger** [comment]
Brief description added to github 'Description' section. I will add a more full description to the readme soon.

**Comment by keiranmraine** [comment]
@drjsanger add to the associated Wiki so that releases aren't needed for documentation and FAQ updates?
